frontend/src/components/dashboard/MineList.js

```diff
@@ -25,28 +25,44 @@ const columns = [
     title: "Mine Name",
     width: 200,
     dataIndex: "mineName",
-    render: (text, record) => <Link to={router.MINE_SUMMARY.dynamicRoute(record.key)}>{text}</Link>,
+    render: (text, record) => (
+      <Link to={router.MINE_SUMMARY.dynamicRoute(record.key)} className="mine-list__name">{text}</Link>
+    ),
   },
   {
     title: "Mine No.",
     width: 100,
     dataIndex: "mineNo",
+    render: (text, record) => (
+      <div title="Mine Number">
+        {text}
+        {!text && <div>{record.emptyField}</div>}
+      </div>
+    ),
   },
   {
     title: "Operational Status",
-    dataIndex: "operationalStatus",
     width: 150,
+    dataIndex: "operationalStatus",
+    render: (text, record) => (
+      <div title="Operational Status">
+        {text}
+        {!text && <div>{record.emptyField}</div>}
+      </div>
+    ),
   },
   {
     title: "Permit No.",
     dataIndex: "permit",
     width: 150,
     render: (text, record) => (
-      <div>
-        {text &&
-          uniqBy(text, "permit_no").map(({ permit_no, permit_guid }) => (
-            <div key={permit_guid}>{permit_no}</div>
-          ))}
+      <div title="Permit Number">
+        <ul className="mine-list__permits">
+          {text &&
+            uniqBy(text, "permit_no").map(({ permit_no, permit_guid }) => (
+              <li key={permit_guid}>{permit_no}</li>
+            ))}
+        </ul>
         {!text && <div>{record.emptyField}</div>}
       </div>
```

Comment:
The alignment on the 'N/A' is off if there are no permits

Reply:
Updated